hunting_season.conf

```
@@ -0,0 +1,13 @@
+name = Hunting season
+author = solars, Peters and -sniper-
```

Actually, solars asked for this line: (https://forum.minetest.net/viewtopic.php?f=12&t=6642&start=275#p341671)

```suggestion
author = solars, Peters and -sniper- from the minetest.net forum
```